docs/ldml/tr35-general.md

```diff
@@ -1477,7 +1477,7 @@ nostr "no:n"
 The generated yesexpr and noexpr would be:
 
 ```
-yesexpr "^([yY]([eE][sS])?)"
+yesexpr "^([yY] ([eE][sS])?)"
```

Comment:
Is the added space intentional?

Reply:
Yes. We can't otherwise easily tell that this isn't a link in markdown, eg

[anchor](relative-link)

Could use a ZWSP so the appearance is better for people used to regex with significant spaces. What do you think?

Reply:
it's inside a code fence - triple backquotes.  A markdown parser can tell the difference.